### DISCUSSION TITLE
[Feature Request] Allow github-actions[bot] avatar to be customized in GitHub Enterprise

### DISCUSSION THREAD

**Discussion by sasha-arrabi** [community/community#146473] [Actions]
### Select Topic Area

General

### Body

In GitHub Enterprise, there doesn't appear to be a way to change the avatar for the `github-actions[bot]` user.

When I hit the `/users/github-actions%5Bbot%5D` API endpoint, I get back an `avatar_url` that points to `/u/2?`, but there doesn't appear to be a way to customize this avatar.

It would be great if we had a way to change the avatar - today, the avatar ends up looking like one of the blocky generic user avatars. We can have a better user experience and branding if we're able to have a custom avatar setup for the `github-actions[bot]` user.

**Comment by lancewillett** [comment]
+1 to customizing the "mystery robot" style default avatars.

**Comment by tylerharris82836-netizen** [comment]
A customizable avatar for github-actions[bot] would definitely improve workflow visibility and branding, especially for teams managing multiple automated pipelines. Features like this, much like a [free company lookup](https://lemreveal.com/data), add valuable transparency and identity to enterprise environments.
